go.mod

```diff
@@ -9,15 +9,16 @@ require (
 	github.com/onsi/gomega v1.20.1
 	github.com/prometheus/client_golang v1.12.2
 	golang.org/x/crypto v0.0.0-20220315160706-3147a52a75dd
-	k8s.io/api v0.25.6
-	k8s.io/apimachinery v0.25.6
-	k8s.io/apiserver v0.25.6
-	k8s.io/client-go v0.25.6
-	k8s.io/code-generator v0.25.6
+	k8s.io/api v0.25.7
+	k8s.io/apimachinery v0.25.7
+	k8s.io/apiserver v0.25.7
+	k8s.io/client-go v0.25.7
+	k8s.io/code-generator v0.25.7
 	k8s.io/klog v1.0.0
 	k8s.io/kube-openapi v0.0.0-20230109183929-3758b55a6596
 	k8s.io/utils v0.0.0-20220728103510-ee6ede2d64ed
 	sigs.k8s.io/controller-runtime v0.13.1
+	sigs.k8s.io/scheduler-plugins v0.25.7
```

Comment:
We use the newest version since this version introduces a new API group, `scheduling.x-k8s.io`.

Reply:
there is no 0.26?

Reply:
Yes, the newest version is 0.25. 
Also, according to https://github.com/kubernetes-sigs/scheduler-plugins/issues/503, the scheduler-plugins seems to be released every 3 months.
So 0.26 will be released in June.

> Yes, we do. Usually, we cut official releases every ~3 months